docs/GettingStartedDocs/advancedBuildInfo.md

```diff
@@ -63,3 +63,16 @@ Building from within a subtree of the build-tree builds all dependencies for tha
 A successful build only outputs the HTML API reference into the build-tree.
 To update the Doxygen-generated documentation published to https://microsoft.github.io/openenclave,
 please follow instructions [here](/docs/refman/doxygen-howto.md)
+
+## Visualizing the CMake Dependency Graph
+
+This is how to use the CMake `graphviz` [support](https://cmake.org/cmake/help/latest/module/CMakeGraphVizOptions.html):
```

Comment:
Consider explaining why we would use graphviz specifically: "CMake comes with built-in [support for `graphviz`](https://cmake.org/cmake/help/latest/module/CMakeGraphVizOptions.html) which makes it easy to generate an image of the dependency graph for a CMake project. For example, to visualize the entire Open Enclave project from the CMake root:"